specification/resources/resource-manager/Microsoft.Resources/stable/2022-12-01/subscriptions.json

```diff
@@ -0,0 +1,874 @@
+            "schema": {
+              "$ref": "#/definitions/CheckZonePeersRequest"
+            },
+            "description": "Parameters for checking zone peers."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns information about the logical availability zone mapping between subscriptions",
+            "schema": {
+              "$ref": "#/definitions/CheckZonePeersResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetLogicalZoneMapping": {
+            "$ref": "./examples/PostCheckZonePeers.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Resources/checkResourceName": {
+      "post": {
+        "tags": [
+          "Tenants"
+        ],
+        "operationId": "checkResourceName",
+        "description": "A resource name is valid if it is not a reserved word, does not contains a reserved word and does not start with a reserved word",
+        "summary": "Checks resource name validity",
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/CheckResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns status as allowed or not.",
+            "schema": {
+              "$ref": "#/definitions/CheckResourceNameResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "CheckValidityForAResourceName": {
+            "$ref": "./examples/CheckResourceName.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PairedRegion": {
+      "description": "Information regarding paired region.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the paired region."
+        },
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The fully qualified ID of the location. For example, /subscriptions/00000000-0000-0000-0000-000000000000/locations/westus."
+        },
+        "subscriptionId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The subscription ID."
+        }
+      }
+    },
+    "AvailabilityZoneMappings": {
+      "description": "Availability zone mappings for the region",
```

Comment:
AvailabilityZoneMappings doesn't have type property defined. It should have "type":"object" property.

Reply:
added